utilities/decrypt_data/decrypt_object.py

```
@@ -0,0 +1,112 @@
+import os
+import json
```

Flake8 (7.2.0)

[error] 'json' imported but unused

(F401)